docs/postprocessors.rst

```diff
@@ -140,6 +145,57 @@ we can see that the best model for the amount of allocated memory depending on c
 
 .. image:: /../examples/clusterizer-memory-scatter-with-models.*
 
+.. _postprocessors-regressogram:
+
+Regressogram method
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: perun.postprocess.regressogram
+
+.. _postprocessors-regressogram-cli:
+
+Command Line Interface
+""""""""""""""""""""""
+
+.. click:: perun.postprocess.regressogram.run:regressogram
+   :prog: perun postprocessby regressogram
+
+.. _postprocessors-regressogram-examples:
+
+Examples
+""""""""
+
+    .. code-block:: json
+
+        {
+            "bin_stats": [
```

Comment:
Not fond of: I think the "bin" is rather ambiguous description (clashes with "binary" used in many places). I understand you mean the window bin, but I'm wondering how we can change this to something different. After some thinking , how about using "bucket" instead? The same upplies to line 176

Reply:
I have replaced all uses of 'bin/bins' for 'bucket/buckets', to make it unambiguous.